src/components/PurchasePage/Order/index.tsx

```diff
@@ -14,23 +39,37 @@ function Order(): React.ReactElement {
           Order
         </Typography>
         <Box>
-          <Button className={classes.clear}>Clear all</Button>
+          <Button className={classes.clear} onClick={() => clearAllProductsAction()}>
```

Comment:
More an FYI, than a request, but if you wanted to with handlers like this, you can pass just the reference to the function without the invocation part:

```suggestion
          <Button className={classes.clear} onClick={clearAllProductsAction}>
```